### DISCUSSION TITLE
Welcome to Smiley Player Trader Discussions!

### DISCUSSION THREAD

**Discussion by Mrcomputer1** [Mrcomputer1/SmileyPlayerTrader#5] [General]
## 👋 Welcome!
We have GitHub Discussions for:
* General discussion about the plugin that isn't related to a issue or pull request.
* Feature requests.
* Questions.

You can also discuss the plugin and ask questions on our Discord server, https://discord.gg/SdM6f7U.

When to use issues and not discussions:
* Tracking a feature that is being implemented (not feature requests).
* Reporting a bug.

**Comment by Semisol** [comment]
ahh yes

**Reply by Mrcomputer1** [reply]
Oh I forgot to tell you, I enabled GitHub Discussions yesterday 😛.

**Reply by Semisol** [reply]
:P
